sw/device/lib/crypto/include/rsa.h

```diff
@@ -28,22 +28,6 @@ typedef enum rsa_padding {
   kRsaPaddingPss = 0x6b1,
 } rsa_padding_t;
 
-/**
- * Enum to define hash modes for RSA schemes.
- *
- * Aligning with existing hash modes. Values are hardened.
- */
-typedef enum rsa_hash {
-  // SHA2-256 hashing mode for RSA.
-  kRsaHashSha256 = 0x378,
```

Comment:
How peculiar: It looks like the rsa_*_keygen_functest targets aren't getting tested, and they fail to build with this change. Just an FYI!

Reply:
Oh no, thanks for letting me know! I'll look into this. The keygen targets are really slow so they're excluded from CI, but I thought at least the 2048 one ran nightly.